src/odemis/driver/xt_client.py

```diff
@@ -612,6 +624,144 @@ def set_autostigmator(self, channel_name, state):
             self.server._pyroClaimOwnership()
             return self.server.set_autostigmator(channel_name, state)
 
+    def get_pitch(self):
+        """
+        Get the pitch between two beams within the multiprobe pattern.
+
+        Returns
+        -------
+        pitch: float, [um]
+            The distance between two beams of the multiprobe pattern.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_pitch()
+
+    def set_pitch(self, pitch):
+        """
+        Set the pitch between two beams within the multiprobe pattern.
+
+        Returns
+        -------
+        pitch: float, [um]
+            The distance between two beams of the multiprobe pattern.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.set_pitch(pitch)
+
+    def pitch_info(self):
+        """"Returns the unit and range of the pitch."""
```

Comment:
In the tests, you explicitly call `self.microscope.pitch_info()["range"]`. So is it a dict that is returned with specific keys "unit" and "range"? Might be worse to extend the doc string. Same for the other "info" methods.

Reply:
I think it's is out of scope for this PR to do it for all info methods. I will do it for pitch and the stigmators and update the others in a new PR.